packages/block-library/src/table/block.json

```diff
@@ -125,6 +122,10 @@
 	"supports": {
 		"anchor": true,
 		"align": true,
+		"color": {
```

Comment:
I've tested it and the background color works as in `trunk`. That part is ready to land in my view. 

However, I'm not sure about text color for this block: it affects both text and the table borders. Shouldn't those things be configurable separately?

One thing we could do to land the parts of this PR that are ready is this: only enable background (and gradients) for now and prepare a follow-up to explore the relationship between text & border gathering more design feedback.

Reply:
> However, I'm not sure about text color for this block: it affects both text and the table borders. Shouldn't those things be configurable separately?

This comes from the active theme. Several themes set the border color to `initial` which means they get set to `currentcolor` and take on the font color. With the new border color block support it could be configurable separately. Which as you say, is better done in a follow up PR.

I thought the greater concern is ensuring sufficient contrast between the background and text colors. If we adopt the colors block support for the background but not text color, do we just create a bigger issue? Is that acceptable if we are looking to dial back this PR to only update background color?

Reply:
Thought about this more: the current behavior is also text + border, so we should be fine.

I guess what I'm uncomfortable with is that this block has a larger design surface than many other blocks and it doesn't look super great at the moment (two background colors -with stripes-, border & text, etc). These are things that Dan brought up already https://github.com/WordPress/gutenberg/pull/30791#issuecomment-823815094 so let's merge and iterate.

Reply:
Sounds like a plan 👍 

I'll merge and take a look at some options to address https://github.com/WordPress/gutenberg/pull/30791#issuecomment-823815094.